src/pkg/cluster/tunnel.go

```diff
@@ -530,3 +530,23 @@ func (tunnel *Tunnel) getAttachablePodForService(ctx context.Context) (string, e
 	}
 	return podList.Items[0].Name, nil
 }
+
+// taken from https://github.com/kubernetes/kubernetes/blob/680ea07dbb2c6050d13b93660fa4d27d2d28d6eb/staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go#L139-L156
+func createDialer(method string, url *url.URL, config *rest.Config) (httpstream.Dialer, error) {
+	transport, upgrader, err := spdy.RoundTripperFor(config)
+	if err != nil {
+		return nil, err
+	}
+	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, method, url)
+	if !cmdutil.PortForwardWebsockets.IsDisabled() {
```

Comment:
I'd drop this if, since you always want to have that full fallback mechanism enabled. At least for quite a few future releases until k8s decides entirely to drop that.

Reply:
Yeah I wasn't sure if there was some niche use case where a user would still want to force the old port forwarder, assuming not I'll go ahead and remove this